src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -58,6 +60,11 @@ public PersonCard(Person person, int displayedIndex) {
         person.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        person.getLabs()
+                .asUnmodifiableObservableList().stream()
+                .sorted(Comparator.comparing(lab -> lab.labNumber))
```

Comment:
Maybe after edit labs and add labs should autosort the lab list. Then the current equality checks also can keep